library/Reporting/Web/Forms/SendForm.php

```diff
@@ -32,8 +35,35 @@ protected function assemble()
 
         (new SendMail())->initConfigForm($this, $this->report);
 
+        $radio = $this->addElement('radio', 'source_radio', [
```

Comment:
`$this->addElement()` doesn't return the newly added element. It returns `$this`, and that's not what you expect, I suppose.